.github/workflows/ci.yml

```diff
@@ -18,9 +18,32 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - id: tag
         uses: ./
         with:
           package-yaml: test/package.yaml
           tag-prefix: example-
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - id: expected
```

Comment:
This "using github actions as your testing tool" is kind of neat.

Reply:
Heh, that's one world for it ;)